notifications/src/main/java/org/openedx/notifications/data/model/NotificaitonCountResponse.kt

```diff
@@ -0,0 +1,23 @@
+package org.openedx.notifications.data.model
+
+import com.google.gson.annotations.SerializedName
+import org.openedx.notifications.domain.model.NotificationsCount
+
+data class NotificationsCountResponse(
+    @SerializedName("show_notifications_tray")
+    var showNotificationsTray: Boolean,
+    @SerializedName("count")
+    var count: Int,
+    @SerializedName("count_by_app_name")
+    var countByAppName: CountByAppNameModel,
+    @SerializedName("notification_expiry_days")
+    var notificationExpiryDays: Int,
+    @SerializedName("is_new_notification_view_enabled")
+    var isNewNotificationViewEnabled: Boolean,
```

Comment:
We should remove the params that we won't need. We can always add them again when needed in the future. After that maybe you can move both models in the same file. :D

Reply:
In my opinion, it's better to retain the response properties within the data model to accurately replicate the API response, and the required properties can then be utilized with the domain models.